src/components/usertask-component/form-fields/number-form-field.tsx

```diff
@@ -48,7 +48,7 @@ export class NumberFormField implements IFormField {
   private _handleKeyDown(event: any): void {
```

Comment:
```suggestion
  private _handleKeyDown(event: KeyEvent): void {
```

or something like this

Reply:
I do not find the right type. 
KeyEvent, KeyboardEvent, Event does not work without rewrote everything with `event.*`

Reply:
```
  private _handleKeyDown(event: KeyboardEvent): void {
    const value: string = this.value + event.key;
    const eventTarget = (event.target as HTMLInputElement);

    const isValidInput: boolean = (!this._inputValidator.validateKey(event.keyCode, eventTarget.value) && !this._inputValidator.isValid(value));

    if (isValidInput) {
      event.preventDefault();
    }
  }
```

Reply:
```suggestion
  private _handleKeyDown(event: KeyboardEvent): void {
```